### DISCUSSION TITLE
An error occurred (TargetNotConnectedException) when calling the ExecuteCommand operation: The execute command failed due to an internal error. Try again later.

### DISCUSSION THREAD

**Discussion by panemsa** [haidaraM/terraform-jenkins-aws-fargate#8] [Q&A]
An error occurred (TargetNotConnectedException) when calling the ExecuteCommand operation: The execute command failed due to an internal error. Try again later.

This repo does not include enable execute command , Can you please help fix it.

**Comment by haidaraM** [comment]
Hi. Which task are you trying to connect to which execute command? Curious to understand the use case before adding the missing permissions
